Source/Android/app/src/main/java/org/dolphinemu/dolphinemu/fragments/ConvertFragment.java

```diff
@@ -206,47 +236,36 @@ public void onStop()
     joinThread();
   }
 
-  private Spinner populateSpinner(int spinnerId, int entriesId, int valuesId,
-          SpinnerValue valueWrapper)
+  private void populateDropdown(TextInputLayout layout, MaterialAutoCompleteTextView dropdown,
+          int entriesId, int valuesId, DropdownValue valueWrapper)
   {
-    Spinner spinner = requireView().findViewById(spinnerId);
-
     ArrayAdapter<CharSequence> adapter = ArrayAdapter.createFromResource(requireContext(),
-            entriesId, android.R.layout.simple_spinner_item);
-    adapter.setDropDownViewResource(android.R.layout.simple_spinner_dropdown_item);
-    spinner.setAdapter(adapter);
-
-    spinner.setEnabled(spinner.getCount() > 1);
+            entriesId, R.layout.support_simple_spinner_dropdown_item);
+    dropdown.setAdapter(adapter);
 
     valueWrapper.populate(valuesId);
-    valueWrapper.setPosition(spinner.getSelectedItemPosition());
-    spinner.setOnItemSelectedListener(valueWrapper);
+    dropdown.setOnItemClickListener(valueWrapper);
 
-    return spinner;
+    layout.setEnabled(adapter.getCount() > 1);
   }
 
-  private Spinner clearSpinner(int spinnerId, SpinnerValue valueWrapper)
+  private void clearDropdown(TextInputLayout layout, MaterialAutoCompleteTextView dropdown,
+          DropdownValue valueWrapper)
   {
-    Spinner spinner = requireView().findViewById(spinnerId);
-
-    spinner.setAdapter(null);
-
-    spinner.setEnabled(false);
+    dropdown.setAdapter(null);
+    layout.setEnabled(false);
 
     valueWrapper.populate(-1);
-    valueWrapper.setPosition(-1);
-    spinner.setOnItemSelectedListener(valueWrapper);
-
-    return spinner;
+    valueWrapper.setPosition(0);
+    dropdown.setText(null, false);
+    dropdown.setOnItemClickListener(valueWrapper);
   }
 
   private void populateFormats()
   {
-    Spinner spinner = populateSpinner(R.id.spinner_format, R.array.convertFormatEntries,
+    populateDropdown(mFormatLayout, mFormatDropdown, R.array.convertFormatEntries,
             R.array.convertFormatValues, mFormat);
-
-    if (gameFile.getBlobType() == BLOB_TYPE_PLAIN)
-      spinner.setSelection(spinner.getCount() - 1);
```

Comment:
Did this disappear from the new code? The intent is to preselect RVZ if the current file type is ISO (since there's no reason to convert from ISO to ISO).